airflow/docker.go

```diff
@@ -209,8 +210,10 @@ func (d *DockerCompose) Start(imageName string, noCache, noBrowser bool) error {
 	airflowDockerVersion := defaultAirflowVersion
 	airflowVersion, ok := imageLabels[airflowVersionLabelName]
 	if ok {
-		if version := semver.MustParse(airflowVersion); version != nil {
+		if version, err := semver.NewVersion(airflowVersion); err == nil {
```

Comment:
@neel-astro please add/update existing tests.

Reply:
added/updated tests to cover the changes